Applications/DisplayProperties/DisplayProperties.cpp

```diff
@@ -281,6 +287,8 @@ void DisplayPropertiesWidget::load_current_settings()
             m_wallpaper_combo->set_text(selected_wallpaper);
             m_wallpaper_combo->set_only_allow_values_from_model(true);
         }
+    }else{
```

Comment:
pls `clang-format`

Reply:
Done! It slipped in the commit, sorry about that.

Reply:
Np!